plugins/nodes/src/js/data/NetworkNodesClient.ts

```diff
@@ -0,0 +1,32 @@
+import { Observable } from "rxjs";
+import { request, RequestResponse } from "@dcos/http-service";
+import { map } from "rxjs/operators";
+
+export interface NetworkNode {
+  updated: string;
+  public_ips: string[];
+  private_ip: string;
+  hostname: string;
+}
+
+export type NetworkNodesResponse = NetworkNode[];
+
+export function fetchNetworkNodes(): Observable<RequestResponse<string[]>> {
+  return request("/net/v1/nodes").pipe(
```

Comment:
it seems like some clients are handling error, some not. 🤔 

https://github.com/dcos/dcos-ui/pull/3578/files#diff-f59206f3b2dceb09dec7ea908fc4fd32R15